__tests__/api-paths.test.js

```diff
@@ -130,18 +130,16 @@ describe("GET /api/reviews/:review_id", () => {
         expect(result.length).toBe(1);
       });
   });
+  test("status:404 when serching for a missing id in reviews", () => {
+    return request(app).get("/api/reviews/92").expect(404);
+  });
+  test("status:400 when serching for an invalid id", () => {
+    return request(app).get("/api/reviews/banana").expect(400);
```

Comment:
this is really good! Only other thing I would suggest is checking that the `msg` you get back is the `msg` you expect - for example in your error handler I see you're sending back a `msg` of `"Bad Path"` for the `400` - however that should say `"Bad Request"`